web/js/containers/sidebar/layer-row.js

```diff
@@ -149,58 +167,68 @@ function LayerRow (props) {
     e.preventDefault();
   };
 
-  const renderControls = () => {
-    const { title } = names;
-    const removeLayerBtnId = `close-${compareState}${encodedLayerId}`;
-    const removeLayerBtnTitle = 'Remove layer';
-
-    const layerOptionsBtnId = `layer-options-btn-${encodedLayerId}`;
-    const layerOptionsBtnTitle = 'View options';
-
-    const layerInfoBtnId = `layer-info-btn-${encodedLayerId}`;
-    const layerInfoBtnTitle = 'View description';
-
-    return (
-      <>
-        <a
-          id={removeLayerBtnId}
-          aria-label={removeLayerBtnTitle}
-          className="button wv-layers-close"
-          onClick={() => onRemoveClick(layer.id)}
-        >
-          <UncontrolledTooltip placement="top" target={removeLayerBtnId}>
-            {removeLayerBtnTitle}
-          </UncontrolledTooltip>
-          <FontAwesomeIcon icon="times" fixedWidth />
-        </a>
-        <a
-          id={layerOptionsBtnId}
-          aria-label={layerOptionsBtnTitle}
-          className={isMobile ? 'hidden wv-layers-options' : 'button wv-layers-options'}
-          onMouseDown={stopPropagation}
-          onClick={() => onOptionsClick(layer, title)}
-        >
-          <UncontrolledTooltip placement="top" target={layerOptionsBtnId}>
-            {layerOptionsBtnTitle}
-          </UncontrolledTooltip>
-          <FontAwesomeIcon icon="sliders-h" className="wv-layers-options-icon" />
-        </a>
-        <a
-          id={layerInfoBtnId}
-          aria-label={layerInfoBtnTitle}
-          className={isMobile ? 'hidden wv-layers-info' : 'button wv-layers-info'}
-          onMouseDown={stopPropagation}
-          onClick={() => onInfoClick(layer, title, measurementDescriptionPath)}
-        >
-          <UncontrolledTooltip placement="top" target={layerInfoBtnId}>
-            {layerInfoBtnTitle}
-          </UncontrolledTooltip>
-          <FontAwesomeIcon icon="info" className="wv-layers-info-icon" />
-        </a>
-      </>
-    );
-  };
+  const renderDropdownMenu = () => (
+    <Dropdown className="layer-group-more-options" isOpen={showDropdownMenu} toggle={toggleDropdownMenuVisible}>
+      <DropdownToggle>
+        <FontAwesomeIcon
+          className="layer-group-more"
+          icon="ellipsis-v"
+        />
+      </DropdownToggle>
+      <DropdownMenu positionFixed>
+        <DropdownItem id={removeLayerBtnId} onClick={() => onRemoveClick(layer.id)}>
```

Comment:
Interacting with this a bit on my phone, I think we could make these dropdown menu items larger across the board for easier interaction.

Reply:
Addressed in c041aca3c